File: src/main/java/com/p2p/lending/dao/BorrowcordDao.java
package com.p2p.lending.dao;

import java.util.List;

import com.p2p.lending.pojo.Borrowcord;

public interface BorrowcordDao {
	
	public List<Borrowcord> selborr(Integer wid);
	
	
	public void updborr(Integer wid);
	
	
	public void borradd(Borrowcord borrowcord);

	
}

File: src/main/java/com/p2p/lending/dao/BorrowcordDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Borrowcord

interface BorrowcordDao {

    fun selborr(wid: Int?): List<Borrowcord>


    fun updborr(wid: Int?)


    fun borradd(borrowcord: Borrowcord)


}
